app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class FileCopyError(Exception):
+    """Raise when my source and target file are the same"""
+
+
+class WrongCommandError(Exception):
+    """Raise when user written wrong command name"""
+
+
+class WrongArgumentsNumberError(Exception):
+    """Raise when user command consists not from 3 elements"""
+
+
+class WrongCommandLengthError(Exception):
+    """Raise when user command have less than 6 symbols"""
+
+
+def copy_file(command: str) -> None:
+    if len(command) < 6:
```

Comment:
Since you are validating that the command contains exactly three parts by splitting the string and checking its length, additional validation of the overall command length is unnecessary. This three-part check is more relevant for your purpose, as it directly ensures that the command follows the expected format, regardless of the individual lengths of the command and file names.